Tests/SwiftLintFramework/IntegrationTests.swift

```diff
@@ -51,6 +51,7 @@ extension String {
             let rawPointer = $0._rawValue
             let byteSize = lengthOfBytesUsingEncoding(NSUTF8StringEncoding)._builtinWordValue
             let isASCII = true._getBuiltinLogicValue()
+            // swiftlint:disable:next variable_name
```

Comment:
I got `variable_name` violation several time about `_builtinStringLiteral` on `Run SwiftLint` build phase and  `IntegrationTests.testSwiftLintLints`.
But, now I can't reproduce that after doing `Clean` on Xcode. 🤔
I'll remove this.